eZ/Publish/Core/Repository/Tests/Service/Integration/RoleBase.php

```diff
@@ -345,21 +345,21 @@ public function testUpdateRole()
         self::assertInstanceOf('\\eZ\\Publish\\API\\Repository\\Values\\User\\Role', $updatedRole);
 
         // @todo: enable
-/*
-        self::assertEquals(
-            array(
-                'eng-US' => $roleUpdateStruct->names['eng-US']
-            ),
-            $updatedRole->getNames()
-        );
-
-        self::assertEquals(
-            array(
-                'eng-US' => $roleUpdateStruct->descriptions['eng-US']
-            ),
-            $updatedRole->getDescriptions()
-        );
-*/
+        /*
+                self::assertEquals(
+                    array(
+                        'eng-US' => $roleUpdateStruct->names['eng-US']
+                    ),
+                    $updatedRole->getNames()
+                );
+
+                self::assertEquals(
+                    array(
+                        'eng-US' => $roleUpdateStruct->descriptions['eng-US']
+                    ),
+                    $updatedRole->getDescriptions()
+                );
+        */
```

Comment:
Code indentation as well
Note to self: commented-out, looks like some follow-up for the future